test/lib-test/jdk/test/lib/hexdump/ASN1FormatterTest.java

```diff
@@ -39,7 +39,7 @@
 /*
  * @test
  * @summary ASN.1 formatting
- * @modules java.base/sun.security.util
+ * @modules java.base/sun.security.util:open
```

Comment:
Roger has since fixed the test via JDK-8255394 so it no longer requires this package to be open.

Reply:
Thanks! I've removed that :open.